packages/@react-spectrum/sidenav/package.json

```diff
@@ -45,7 +45,7 @@
     "@react-stately/collections": "^3.1.0",
     "@react-stately/layout": "^3.2.1",
     "@react-stately/tree": "^3.1.0",
-    "@react-stately/virtualizer": "3.1.7-alpha.0",
+    "@react-stately/virtualizer": "^3.1.7",
```

Comment:
it should go back to ^3.1.0 i think

Reply:
I changed it to `^3.1.7` since all the other packages that had this same dep got bumped to the same version

Reply:
ah, ok, what'd we add that they all need suddenly?

Reply:
grid looks in trouble too? https://github.com/adobe/react-spectrum/commit/dd5d61a74bb570187f236bf2fb3adb12267e0a4f#diff-4292ce431eb32c342ae6db59c8e48a69ecc9f6389a72d7052586a2e91c50e180L29

Reply:
resolved, fixed in patch publish update